src/lib/components/tablev2/Tablev2.component.tsx

```diff
@@ -42,6 +42,7 @@ export type TableProps<
   globalFilter?: string;
   onBottom?: (rowLength: number) => void;
   onBottomOffset?: number;
+  allFilters?: { id: string; value: string }[];
```

Comment:
Can you remind me the context where this is necessary ? Thinking wether we could come up with a better abstraction

Reply:
The context is filtering by a single (or in that matter multiple) column. Global filter filters by occurrence of a string in all columns, in the case of buckets and endpoints tables, we need to filter by a single column. The prop name and usage is one to one with react-table, my idea was to make it as familiar as possible to people who have used it. And to be able to easily see documentation about its usage